<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QMainWindow" name="Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Maze Maker</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	font: 14pt &quot;Times New Roman&quot;;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>600</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="menu">
       <widget class="QPushButton" name="btn_load">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>400</y>
          <width>300</width>
          <height>50</height>
         </rect>
        </property>
        <property name="text">
         <string>Загрузить лабиринт</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="line_width">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>75</y>
          <width>100</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>10</string>
        </property>
        <property name="maxLength">
         <number>3</number>
        </property>
       </widget>
       <widget class="QLineEdit" name="chance_b_wall">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>225</y>
          <width>40</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>50</string>
        </property>
        <property name="maxLength">
         <number>3</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>20</y>
          <width>300</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Настройка параметров</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="chance_r_wall">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>175</y>
          <width>40</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>50</string>
        </property>
        <property name="maxLength">
         <number>3</number>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="line_height">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>125</y>
          <width>100</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>10</string>
        </property>
        <property name="maxLength">
         <number>3</number>
        </property>
       </widget>
       <widget class="QTextEdit" name="field_errors">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>300</y>
          <width>600</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(234, 234, 234);</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Times New Roman'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>75</y>
          <width>100</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Ширина</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>175</y>
          <width>250</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Вероятность правых стен</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>125</y>
          <width>100</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Высота</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>225</y>
          <width>250</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Вероятность нижних стен</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_start">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>400</y>
          <width>300</width>
          <height>50</height>
         </rect>
        </property>
        <property name="text">
         <string>Старт</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="maze">
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="4" column="3">
         <widget class="QPushButton" name="btn_save_img">
          <property name="text">
           <string>Сохранить в jpg/png</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="btn_menu">
          <property name="text">
           <string>Меню</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="4">
         <widget class="QGraphicsView" name="view">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="backgroundBrush">
           <brush brushstyle="NoBrush">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </property>
          <property name="viewportUpdateMode">
           <enum>QGraphicsView::MinimalViewportUpdate</enum>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="btn_find_way">
          <property name="text">
           <string>Запуск волны</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="btn_save_txt">
          <property name="text">
           <string>Сохранить в txt</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
